Xamarin.Essentials/Contacts/Contacts.tizen.cs

```diff
@@ -0,0 +1,130 @@
+using TizenEmail = Tizen.Pims.Contacts.ContactsViews.Email;
+using TizenName = Tizen.Pims.Contacts.ContactsViews.Name;
+using TizenNumber = Tizen.Pims.Contacts.ContactsViews.Number;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Contacts
+    {
+        static async Task<Contact> PlatformPickContactAsync()
+        {
+            Permissions.EnsureDeclared<Permissions.ContactsRead>();
+            await Permissions.EnsureGrantedAsync<Permissions.ContactsRead>();
+
+            var tcs = new TaskCompletionSource<Contact>();
+
+            var appControl = new AppControl();
+            appControl.Operation = AppControlOperations.Pick;
+            appControl.ExtraData.Add(AppControlData.SectionMode, "single");
+            appControl.LaunchMode = AppControlLaunchMode.Single;
+            appControl.Mime = "application/vnd.tizen.contact";
+
+            AppControl.SendLaunchRequest(appControl, (request, reply, result) =>
+            {
+                Contact contact = null;
+
+                if (result == AppControlReplyResult.Succeeded)
+                {
+                    var contactId = reply.ExtraData.Get<IEnumerable<string>>(AppControlData.Selected)?.FirstOrDefault();
+
+                    if (int.TryParse(contactId, out var contactInt))
+                    {
+                        var mgr = new ContactsManager();
+
+                        var record = mgr.Database.Get(TizenContact.Uri, contactInt);
+
+                        if (record != null)
+                        {
+                            string name = null;
+                            var emails = new List<ContactEmail>();
+                            var phones = new List<ContactPhone>();
+
+                            var recordName = record.GetChildRecord(TizenContact.Name, 0);
+                            if (recordName != null)
+                            {
+                                var first = recordName.Get<string>(TizenName.First) ?? string.Empty;
+                                var last = recordName.Get<string>(TizenName.Last) ?? string.Empty;
+
+                                name = $"{first} {last}".Trim();
+                            }
+
+                            var emailCount = record.GetChildRecordCount(TizenContact.Email);
+                            if (emailCount > 0)
+                            {
+                                for (var i = 0; i < emailCount; i++)
+                                {
+                                    var item = record.GetChildRecord(TizenContact.Email, i);
+                                    var addr = item.Get<string>(TizenEmail.Address);
+                                    var type = item.Get<int>(TizenEmail.Type);
+
+                                    var emailType = TizenEmail.Types.Other;
+                                    if (Enum.IsDefined(typeof(TizenEmail.Types), type))
+                                        emailType = (TizenEmail.Types)type;
```

Comment:
Took me a few reads to see that this was basically saying use other if the type is invalid. But, since this is an enum, what are the possible values? Maybe let the `GetContactType` actually do this? If `type` is int, then you can still just cast it to an enum.

```csharp
enum Test {
  A = 0,
  B = 1,
}

int v = 1000;
Test res = (Test)v;
```

that is still valid C# code and then the `_ => ` in the switch will catch it.